tools/code-generation/smithy/codegen/cpp-smoke-tests-codegen/src/main/java/com/amazonaws/util/awsclientsmithygenerator/generators/CppImportContainer.java

```diff
@@ -41,21 +43,19 @@ public CppImportContainer(String namespace) {
             "algorithm"
             );
 
-        this.unitTestHeaders = Set.of(
-            "aws/testing/AwsCppSdkGTestSuite.h",
-            "aws/testing/AwsTestHelpers.h"
-            );
+        this.unitTestHeaders = new HashSet<>();
```

Comment:
Should use guavas `ImmubtableSet.of` and add the [guava dep](https://mvnrepository.com/artifact/com.google.guava/guava) if needed, would be nice but not required